adb/fastboot.py

```diff
@@ -151,24 +152,33 @@ def _AcceptResponses(self, expected_header, info_cb, timeout_ms=None):
           FastbootInvalidResponse: Fastboot responded with an unknown packet type.
 
         Returns:
-          OKAY packet's message.
+          Tuple - OKAY packet's message, List of preceding Fastboot Messages
```

Comment:
What was this change for?  It seems like it just makes it so the INFO messages are now returned as well but could that break people since the return format here is new?  To be honest it's not a huge deal since I'd imagine not a ton of people using FastbootProtocol directly but just curious

Reply:
Yeah I didn't like making this change because of API breakage, but in a few tools I've integrated this with I've found it's useful for callers to have a list of the FastbootMessages that were a part of the response to any SimpleCommands.

info_cb is useful for logging, but having a list of messages returned right after a call is made is good for making decisions based on specific returns from a command.  'OKAY' only returns if the command was successfully run, but does not capture if the command itself had issues (ie. fastboot boot may return 'OKAY', ['oem unlocking is not enabled'] )